SMSEagleIOTBridge/Readme_Prepare_Eagle.md

```diff
@@ -117,167 +96,128 @@ should give
 
-If the repository does not exist anymore for some reason, you can probably find the files on one of the existing SMSEagles in the folder /home/pi. 
-After you have downloaded them, make sure that the filename is exactly that, and e.g. windows did not "accidentally" add .txt as an extension.
-
-Copy the following files to the SMSEagles /home/pi folder:
-```
-nyssIotBridge.py
-smsEagle-iot-hub-handler.py
 ```
-
-Via ssh you can use the following commands:
-```
-scp "C:/path on windows to file" root@ipOfEagle:/home/pi
+cd /home/pi
 ```
 
-### 1.2.4. Copy service file
+2. Download the following scripts with curl
 
-If you installed Python using what is described in Chapter 1.2.2.1., you can skip step 2.
-
-#### 1. Get the necessary files
-You need the service file as well as the shell script to run before the service starts. The files are in the same repository as the python script:
-- https://github.com/nyss-platform-norcross/nyss-sms-gateway
-
-You need the following files:
-* https://github.com/nyss-platform-norcross/nyss-sms-gateway/blob/master/SMSEagleIOTBridge/nyss-iot-bridge.service
-* https://github.com/nyss-platform-norcross/nyss-sms-gateway/blob/master/SMSEagleIOTBridge/nyssIotBridgeBoot.sh
-
-If the repository does not exist anymore for some reason, you can probably find the files on one of the existing SMSEagles in the folder /etc/systemd/system.
-
-#### 2. Adjust path if necessary (only if you installed Python the hard way)
-The path in that file needs to match whereever you have copied the smsEagle-iot-hub-handler.py file (usually that would be /home/pi).
-
-If you had to install Python the hard way **(and only then)**, you need to change the path in the service to reach your python installation. Change the following:
-
-```
-ExecStart=/usr/bin/python3 /home/pi/smsEagle-iot-hub-handler.py
 ```
+curl -o nyssIoTBridge.py https://raw.githubusercontent.com/nyss-platform-norcross/nyss-sms-gateway/master/SMSEagleIOTBridge/nyssIotBridge.py 
 
-to 
-
+curl -o smsEagle-iot-hub-handler.py https://raw.githubusercontent.com/nyss-platform-norcross/nyss-sms-gateway/master/SMSEagleIOTBridge/smsEagle-iot-hub-handler.py
 ```
-ExecStart=/usr/local/bin/python3.6 /home/pi/smsEagle-iot-hub-handler.py
-```
-
-You might have to adjust that path, depending on where your installation is located. You can find that out using the following command:
 
-```
-whereis python3
-```
+### 2.2. Copy service files
 
-#### 3. Copy the files
+1. Enter the correct folder
 
-Copy the following file to the SMSEagles /etc/systemd/system folder:
-```
-nyss-iot-bridge.service
 ```
-And the following file to the SMSEagles /home/pi folder
+cd /etc/systemd/system
 ```
-nyssIotBridgeBoot.sh
-```
-Via ssh you can use the following commands as run from your host computer and not the SMSEagle:
-```
-scp "C:/path on windows to service file" root@ipOfEagle:/etc/systemd/system
-scp "C:/path on windows to shell script file" root@ipOfEagle:/home/pi
+2. Download service files with curl
```

Comment:
This is good to have here, but I would add also the option to run the setup.sh script. The setup script creates all the needed folders and downloads the files to the correct place, which means replacing basically all of this.